docs/guide.rst

```diff
@@ -398,6 +398,41 @@ Sometimes you forget something, or just things may go wrong and
 you need another try. In such case add ``-f`` or ``--force`` to
 quickly overwrite existing files with the right content.
 
+.. _linking_tmt_objects:
+
+Linking tmt objects
+------------------------------------------------------------------
+If you create a Jira issue for a bug or a feature, you can link it
+to the test or plan that covers it. This can be either done automatically
+on creation of a new test or plan, or later on using the ``tmt link`` command.
+
+.. code-block:: shell
+
+    tmt link verifies:issues.redhat.com/browse/YOUR-ISSUE tests/core/smoke
```

Comment:
Why not make schema part of the link from the beginning? Examples at https://tmt.readthedocs.io/en/stable/spec/core.html#link use it like it's nothing to be afraid of. `verifies:https://issues.redhat.com/browse/YOUR-ISSUE` would be complete, explicitly stated by the user, no space for tmt guess the wrong scheme. If I run my own ticket tracking service in my local network, without https, I wouldn't be able to use `tmt link` because it would be injecting `https://` into my links.